### PR TITLE
Fix publiclab/plots2#1208

### PR DESCRIPTION
**Pull request by ryzokuken**
Add functionality to the `read more` links and make them open the link in another tab

### CODE REVIEW

File: src/modules/PublicLab.TagsModule.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = PublicLab.TagsModule = PublicLab.Module.extend({
     _module.key = 'tags';
     _module.options = options || _editor.options.tagsModule || {};
     _module.options.name         = 'tags';
-    _module.options.instructions = 'Tags relate your work to others\' posts. <a href="">Read more &raquo;</a>';
+    _module.options.instructions = 'Tags relate your work to others\' posts. <a href="/post" target="_blank">Read more &raquo;</a>';
```
Comment (jywarren) [inline src/modules/PublicLab.TagsModule.js]: Could we change this URL to `https://publiclab.org/wiki/authoring-help#Tags` and we'll add that section in later?

Comment (ryzokuken) [inline src/modules/PublicLab.TagsModule.js]: Yeah, but should we really use implicit external links?

**Comment by jywarren** [comment]
I think we could have this settable in the settings for this module, with a
default that doesn't include them. So we'd use an in-module settable option
such as options.helpText - how's that sound?

**Comment by jywarren** [comment]
OK -- for speed, i think we can just add `target="_blank"` as a first step to solve the immediate issue, then follow up with a more customizable `module.helpText` feature. How's that sound?

**Comment by ryzokuken** [comment]
@jywarren Sounds great! Already made the appropriate changes. Look into them whenever possible. :D

**Comment by ryzokuken** [comment]
@jywarren Sorry for the (almost) unnecessary whitespace stripping. But I also realized... should we not run a linter for the project? Perhaps we could integrate one in Danger, look for poorly formatted/indented code?

**Comment by jywarren** [comment]
no problem, thanks.

Is a linter part of Coveralls integration @ananyo2012 added? or could it be?

**Comment by jywarren** [comment]
Oh whoops, this is the Editor repo! That was for plots2. But i think that's
reasonable... do you think a project that's smaller like this merits
linting?

**Comment by jywarren** [comment]
Looks great, merging!

**Pull request merged**